oak_containers_orchestrator/src/logging.rs

```diff
@@ -34,7 +34,7 @@ pub fn setup() -> Result<(), Box<dyn std::error::Error>> {
         syslog::unix(formatter).map_err(|e| format!("impossible to connect to syslog: {:?}", e))?;
 
     log::set_boxed_logger(Box::new(BasicLogger::new(logger)))
-        .map(|()| log::set_max_level(LevelFilter::Trace))
+        .map(|()| log::set_max_level(LevelFilter::Debug))
```

Comment:
FYI I think it should also be possible to set logging in a more fine grained way (e.g. per package), though I don't remember how off the top of my head.

Reply:
BTW, should this be something that would be useful to pass in from the outside, e.g. as part of the config?

Reply:
Yes, we should figure out a better API for this.